packages/experimental/assistant/src/ai-service/client.ts

```diff
@@ -12,14 +12,16 @@ import { log } from '@dxos/log';
 import { createGenerationStream, type GenerationStream } from './stream';
 import { type GenerateRequest } from './types';
 
-// TODO(burdon): Rename.
-export interface AIServiceClient {
+export interface AIService {
+  exec(request: GenerateRequest): Promise<GenerationStream>;
+}
+
+export interface AIServiceClient extends AIService {
   getSpace(spaceId: SpaceId): Promise<Space>;
   getThread(spaceId: SpaceId, threadId: string): Promise<Thread>;
   getMessages(spaceId: SpaceId, threadId: string): Promise<Message[]>;
   appendMessages(messages: Message[]): Promise<void>;
   updateMessage(spaceId: SpaceId, threadId: string, messageId: string, message: Message): Promise<void>;
```

Comment:
those CRUD methods are depreacted since we've moved away from storing context in ai-service and moved to queues